ModuleFile

```diff
@@ -1,7 +1,8 @@
 name 'citops-repose'
-version '1.8.2'
+version '2.0.0-1'
 description "Repose is an api middleware that provides authentication,
 filtering, ratelimitting and several other features, this deploys it."
 project_page 'https://github.com/rackerlabs/puppet-repose'
 
 dependency 'puppetlabs/stdlib', '>= 2.0.0'
+dependency 'camptocamp/systemd', '>= 0.4.0'
```

Comment:
if justin's works on p4, then it wins at the moment. go ahead and create an issue suggesting that we re-evaluate camptocamp once that issue is addressed